specification/validated-relationships-service-api.yaml

```diff
@@ -3,7 +3,7 @@
 openapi: '3.0.0'
 info:
   title: 'Validated Relationships Service API'
-  version: '1.1.0'
+  version: '1.2.0'
   description: |
     ## Overview
     Use this API to access the Validated Relationships Service - the national electronic database of relationships
```

Comment:
Line 19:

We're adding an extra operation here, so should move line 23 up into the the "you can" section